build/dev/Dockerfile

```diff
@@ -10,13 +10,10 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM quay.io/eclipse/che-golang-1.12:nightly
+FROM quay.io/eclipse/che-golang-1.14:nightly
```

Comment:
woo, skipping 1.13 and going directly to 1.14! #consistentcy-win!

Reply:
We never even build che-golang-1.13 as far as I know :smile:

Reply:
In this house we only respect even-numbered Go releases: [1.10](https://quay.io/repository/eclipse/che-golang-1.10), [1.12](https://quay.io/repository/eclipse/che-golang-1.12), [1.14](https://quay.io/repository/eclipse/che-golang-1.14)